frontend/src/metabase/browse/components/ModelsTable.tsx

```diff
@@ -126,9 +147,20 @@ export const ModelsTable = ({ models }: ModelsTableProps) => {
         </tr>
       </thead>
       <TBody>
-        {sortedModels.map((model: ModelResult) => (
-          <TBodyRow model={model} key={`${model.model}-${model.id}`} />
-        ))}
+        {error || isLoading ? (
+          <LoadingAndErrorWrapperTableRow>
+            <td colSpan={5}>
+              <DelayedLoadingAndErrorWrapper
+                error={error}
+                loading={!!isLoading}
```

Comment:
nested loading components is a bit confusing 🥴

Reply:
I'm not seeing the nested loading components? 🤔 

`LoadingAndErrorWrapperTableRow` isn't a loading component fwiw - it's a style component that wraps the `DelayedLoadingAndErrorWrapper`